### PR TITLE
Split failpoints file

### PR DESCRIPTION
**Pull request by serathius**
Should reduce number of code conflicts and allow better scaling.

### CODE REVIEW

File: tests/robustness/failpoint/gofail.go [hunk 1]
```diff
@@ -0,0 +1,189 @@
...
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package failpoint
+
+import (
+	"context"
+	"fmt"
+	"go.etcd.io/etcd/tests/v3/framework/e2e"
```
Comment (fuweid) [inline tests/robustness/failpoint/gofail.go:20]: need to move this one into other imported group

Comment (serathius) [reply tests/robustness/failpoint/gofail.go]: Thanks :P

**Comment by serathius** [comment]
cc @ahrtr

**Review by fuweid — APPROVED** [review]
LGTM

**Pull request merged**